monkey/tests/unit_tests/conftest.py

```diff
@@ -50,3 +51,17 @@ def monkey_configs_dir(data_for_tests_dir) -> Path:
 @pytest.fixture
 def default_agent_configuration() -> AgentConfiguration:
     return DEFAULT_AGENT_CONFIGURATION
+
+
+@pytest.fixture
+def event_queue_subscriber():
```

Comment:
This isn't necessary. The only thing that needs a real subscriber are the event queue tests. Everything else that publishes can receive a mock `IEventQueue` and you can validate that what is published is correct. See also the comments on `test_mimikatz_collector.py`.